src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      action="/big-search"
+      method="POST"
+      data-qa="big"
+    >
+      <img
+        class="search-icon"
+        src="./images/Search.svg"
+        alt="Search Icon"
+      >
+      <label for="input">
+        <input
```

Comment:
Also add name attribute for inputs but for every input this name should be unique